k8s/examples/kafka/01-rbac.yaml

```diff
@@ -0,0 +1,38 @@
+---
+# Source: kafka/templates/rbac.yaml
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kafka
+  namespace: default
```

Comment:
The namespace definition blocks the user to be namespace agnostic. 
Could we please remove the `namespace: default`.

Reply:
removed